packages/cli/src/commands/capabilities/translations.ts

```diff
@@ -2,9 +2,9 @@ import { flags } from '@oclif/command'
 
 import {CapabilityLocalization, LocaleReference} from '@smartthings/core-sdk'
 
-import { APICommand, NestedListingOutputAPICommandBase, stringTranslateToNestedId } from '@smartthings/cli-lib'
+import { APICommand, ListCallback, NestedListingOutputAPICommandBase, stringTranslateToNestedId } from '@smartthings/cli-lib'
 
-import { capabilityIdOrIndexInputArgs, getCustomByNamespace, translateToId,
+import { capabilityIdOrIndexInputArgs, getCustomByNamespace, translateToId as translateToId,
```

Comment:
Is this rename actually serving a purpose?

Reply:
Um, no. 😄  Not sure how that got in there...I had temporarily renamed this function to `xxx_translateToId` using F2 in VSCode so I could easily separate it from any other functions of the same name and VSCode must have put this is there when I did that. I'll remove it before I merge.